lua/hbac/telescope/init.lua

```diff
@@ -29,9 +23,21 @@ local attach_mappings = function(opts)
 	end
 end
 
+local parse_opts = function(opts)
+	local hbac_telescope = require("hbac.telescope")
+	if not hbac_telescope then
+		return
+	end
```

Comment:
Does this ever return anything other than true? We had these lines in the removed `telescope` subcommand. (`hbac_telescope` would return false if the dependency check removed above would fail). Or is it for something else?

Reply:
No, you are right. We can scratch that.